modules/candidate_parameters/jsx/CandidateInfo.js

```diff
@@ -252,6 +289,22 @@ var CandidateInfo = React.createClass(
               disabled={reasonDisabled}
               required={reasonRequired}
             />
+            <TagsElement
```

Comment:
I feel like this element is similar to the Searchable Dropdown element. Could the 'add Tags' feature just be added to the existing Searchable Dropdown as an optional?

Reply:
i'd rather not. the searchable dropdown is just one aspect of it, if you set `useSearch` to false it becomes a normal select, and if you remove `options` altogether it becomes like an open textbox

Reply:
gotcha